### PR TITLE
Make sensor entry.sh executable

### PR DESCRIPTION
**Pull request by chrisys**
Change-type: patch
Signed-off-by: Chris Crocker-White <chriscw@balena.io>

### CODE REVIEW

File: sensor/Dockerfile.template [hunk 1]
```diff
@@ -19,5 +19,6 @@ RUN pip install smbus
 WORKDIR /usr/src/app
 COPY ./scripts ./scripts
 COPY ./*.sh /usr/src/app/
+RUN chmod +x /usr/src/app/entry.sh
```
Comment (zrzka) [inline sensor/Dockerfile.template]: Can work, but you can omit this line, do `chmod +x ...` on your computer and commit new file attributes only?

Comment (chrisys) [reply sensor/Dockerfile.template]: @zrzka the file already has +x attributes, but in my experience this doesn't cover all cases. Not sure if it's users pushing from Windows or something but without this we always get issues.

Comment (zrzka) [reply sensor/Dockerfile.template]: @chrisys ah, interesting, never had this issue (okay, not using Windows :), thanks for the explanation

Comment (chrisys) [reply sensor/Dockerfile.template]: I personally have never been able to reproduce it (macOS user) but it's come up across all of the projects we promote heavily, might be an edge case.

**Comment by zrzka** [comment]
@chrisys oh, one more thing ...

```
COPY ./*.sh /usr/src/app/
 RUN chmod +x /usr/src/app/entry.sh
```

... just a cosmetic one, but ...

* you're copying all `*.sh` files
* but then you're setting `+x` on one file

... shouldn't we copy just this file or set `+x` on all `*.sh` files then?

**Comment by chrisys** [comment]
@zrzka good point. This is good discussion, I'm going to get you to review more often! 😉

**Comment by zrzka** [comment]
Hehe, feel free @chrisys :)

**Pull request merged**